src/Microsoft.Management.Configuration.UnitTests/Tests/ConfigurationMixedElevationTests.cs

```diff
@@ -0,0 +1,176 @@
+// -----------------------------------------------------------------------------
+// <copyright file="ConfigurationMixedElevationTests.cs" company="Microsoft Corporation">
+//     Copyright (c) Microsoft Corporation. Licensed under the MIT License.
+// </copyright>
+// -----------------------------------------------------------------------------
+
+namespace Microsoft.Management.Configuration.UnitTests.Tests
+{
+    using System;
+    using System.Threading.Tasks;
+    using Microsoft.Management.Configuration.UnitTests.Fixtures;
+    using Microsoft.Management.Configuration.UnitTests.Helpers;
+    using Microsoft.VisualBasic;
+    using Xunit;
+    using Xunit.Abstractions;
+
+    /// <summary>
+    /// Unit tests for verifying the processor behavior for handling mixed elevation scenarios.
+    /// </summary>
+    [Collection("UnitTestCollection")]
+    [OutOfProc]
+    [OutOfProcOnly]

```

Comment:
I assume that these tests still attempt to run if you select everything in the Test Explorer?  I'm not sure how to default them to not selected, but that would be better if we can.

Reply:
I was not able to find a way to have these skipped in Test Explorer. There is an option to do something like Fact[Skip="reason"] but that causes the test to always be skipped.